example/data/collections/L2_HR_PIXC-000/L2_HR_PIXC-000.json

```diff
@@ -8,9 +8,6 @@
   "reportToEms": false,
   "url_path": "{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}",
   "sampleFileName": "L2_HR_PIXC_product_0001-of-4154.h5",
-  "meta": {
-    "excludeFileTypes": [".cmr.json", ".cmr.xml", ".h5", ".hdf", ".jpg", ".md5", ".met", ".nc", ".txt"]
```

Comment:
Why did you get rid of this meta stuff in all these json files?

Reply:
Before this PR, IngestAndPublishGranule workflow has copy_to_glacier step, and excludeFileTypes was added to prevent granule files get copied to glacier.  Now only IngestAndPublishGranuleWithOrca has orca step, and this workflow is invoked in the OrcaBackupAndRecoverySpec.